cmd/consumer/protobuf/antrea.pb

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 VMware, Inc.
```

Comment:
We should probably move the protobuf folder to pkg/consumer.

Reply:
Thought more about this. Maybe we could use the proto code (protobuf and generated code) directly from Antrea repo, so that we will not have any compatibility issues later. What do you think?

Reply:
Will it cause dependency cycle if we import the proto code from Antrea and Antrea import other lib from go-ipfix?

Reply:
I see that antrea proto is only used in cmd/consumer. I do not think we have to import this package to Antrea.
Lets keep it separate for now. We can make the change when Antrea code gets merged later.